src/glue/SessionHL.cpp

```diff
@@ -121,9 +122,14 @@ void MultiDatabaseAuth(memgraph::query::QueryUserOrRole *user, std::string_view
 namespace memgraph::glue {
 
 #ifdef MG_ENTERPRISE
-std::string SessionHL::GetDefaultDB() {
+std::optional<std::string> SessionHL::GetDefaultDB() {
```

Comment:
Future change: do we want stronger default DB? `name + uuid`

Reply:
That makes sense.
I will implement this now, since we missed the 2.17 release.

Reply:
The whole interface is name based. 
The user can define the default database and access to a database via its name.
Drivers that connect directly to a database use the name.
There is no auth logic that handles database UUIDs. It should be transparent to it.
If a database is deleted, it is removed from auth as well.